treatment-service/config/config.go

```diff
@@ -96,8 +95,8 @@ type ManagementServiceConfig struct {
 }
 
 type ManagementServicePollerConfig struct {
-	Enabled      bool          `default:"false"`
-	PollInterval time.Duration `default:"30s"`
+	Enabled             bool `json:"enabled" default:"false"`
+	PollIntervalSeconds int  `json:"poll_interval" default:"30"`
```

Comment:
Addition of `json` tags to the fields of `ManagementServicePollerConfig` and changing of the `PollInterval` field to the `int` data type.